### PR TITLE
create component of a navbar for a suggestion

### PR DESCRIPTION
**Pull request by AlexisFLT**

### CODE REVIEW

File: frontend/src/components/NavBarSuggest/NavBarSuggest.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  return (
+    <div className="navBarS">
+      <button
+        type="button"
+        className="firstB"
+        style={{
+          backgroundColor: isActive === "idea" ? "#70af90" : "#70af8f60",
+          boxShadow: isActive === "idea" ? "0 20px 60px 0 black" : "unset",
+          cursor: "pointer",
+        }}
```
Comment (Sax593) [inline frontend/src/components/NavBarSuggest/NavBarSuggest.jsx]: The style tags are to be avoided. Is it possible to rework this?

File: frontend/src/components/NavBarSuggest/NavBarSuggest.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+      <button
+        type="button"
+        className="thirdB"
+        style={{
+          backgroundColor: isActive === "votes" ? "#70af90" : "#70af8f60",
+          boxShadow: isActive === "votes" ? "0 20px 60px 0 black" : "unset",
+          cursor: "pointer",
+        }}
+        onClick={() => setIsActive("votes")}
```
Comment (Sax593) [inline frontend/src/components/NavBarSuggest/NavBarSuggest.jsx:43]: Same

File: frontend/src/components/NavBarSuggest/NavBarSuggest.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      </button>
+
+      <button
+        type="button"
+        className="fourthB"
+        style={{
+          backgroundColor: isActive === "finaleD" ? "#70af90" : "#70af8f60",
+          boxShadow: isActive === "finaleD" ? "0 20px 60px 0 black" : "unset",
+          cursor: "pointer",
+        }}
```
Comment (Sax593) [inline frontend/src/components/NavBarSuggest/NavBarSuggest.jsx]: Same

File: frontend/src/components/NavBarSuggest/style.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
+@use "../../App.scss";
+
+
```
Comment (Sax593) [inline frontend/src/components/NavBarSuggest/style.scss:4]: unnecessary space

File: frontend/src/components/NavBarSuggest/NavBarSuggest.jsx (hunk 1 above)
Comment (Sax593) [inline frontend/src/components/NavBarSuggest/NavBarSuggest.jsx]: The style tags are to be avoided. Is it possible to rework this?

File: frontend/src/components/NavBarSuggest/NavBarSuggest.jsx (hunk 2 above)
Comment (Sax593) [inline frontend/src/components/NavBarSuggest/NavBarSuggest.jsx:43]: Same

File: frontend/src/components/NavBarSuggest/NavBarSuggest.jsx (hunk 3 above)
Comment (Sax593) [inline frontend/src/components/NavBarSuggest/NavBarSuggest.jsx]: Same

**Review by Sax593 — CHANGES_REQUESTED** [review]

**Review by Sax593 — DISMISSED** [review]

File: frontend/src/components/NavBarSuggest/style.scss (hunk 4 above)
Comment (AlexisFLT) [reply frontend/src/components/NavBarSuggest/style.scss:4]: removed

File: frontend/src/components/NavBarSuggest/NavBarSuggest.jsx (hunk 1 above)
Comment (AlexisFLT) [reply frontend/src/components/NavBarSuggest/NavBarSuggest.jsx]: removed

Comment (AlexisFLT) [reply frontend/src/components/NavBarSuggest/NavBarSuggest.jsx]: removed

Comment (AlexisFLT) [reply frontend/src/components/NavBarSuggest/NavBarSuggest.jsx]: removed

File: frontend/src/components/NavBarSuggest/NavBarSuggest.jsx (hunk 2 above)
Comment (AlexisFLT) [reply frontend/src/components/NavBarSuggest/NavBarSuggest.jsx:43]: removed

File: frontend/src/components/NavBarSuggest/NavBarSuggest.jsx (hunk 3 above)
Comment (AlexisFLT) [reply frontend/src/components/NavBarSuggest/NavBarSuggest.jsx]: removed

File: frontend/src/components/NavBarSuggest/NavBarSuggest.jsx (hunk 2 above)
Comment (AlexisFLT) [reply frontend/src/components/NavBarSuggest/NavBarSuggest.jsx:43]: removed

**Review by Sax593 — APPROVED** [review]

File: frontend/src/components/NavBarSuggest/style.scss [hunk 5]
```diff
@@ -0,0 +1,159 @@
+@use "../../App.scss";
+
+//PAD
+
```
Comment (LoicBrassart) [inline frontend/src/components/NavBarSuggest/style.scss:4]: C'toi l'Pad

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**